3-enrich/stream-enrich/core/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/Amodel.scala

```diff
@@ -64,6 +66,17 @@ object model {
       case cn @ "cn-north-1" => s"https://kinesis.$cn.amazonaws.com.cn"
       case _                 => s"https://kinesis.$region.amazonaws.com"
     })
+
+
+    val dynamodbEndpoint = dynamodbCustomEndpoint.getOrElse(region match {
+      case cn @ "cn-north-1" => s"https://dynamodb.$cn.amazonaws.com.cn"
+      case _                 => s"https://dynamodb.$region.amazonaws.com"
+    })
+
+    val iscloudwatchDisabled = disableCloudWatch.getOrElse(region match {
+      case _                 => false
+    })
```

Comment:
Line 76-78 is not helpful. please remove them

Reply:
This was my attempt at following what I thought was the snowplow convention. When we run this code `val worker = if (kinesisConfig.disableCloudWatch.isDefined && kinesisConfig.iscloudwatchDisabled == true)` we are checking if the value we set in the config file is true. Without these lines of code we would have to check `if kinesisConfig.disableCloudWatch == "some(false)". This is completely fine with me, just letting you know so that we are on the same page

Reply:
you won't need it. That `if condition` there might do.

Reply:
It seems that I was wrong, `if (kinesisConfig.disableCloudWatch.isDefined && kinesisConfig.disableCloudWatch == "some(false)")` does not work. In the debugger it shows that `disableCloudWatch = {Some@7519} "Some(true)"` with the `{Some@7519}` greyed out which is why I ignored it. I tried including it afterwards but then the program fails to compile, not to mention that the number `7519` changes every time I recompile. I will try some more things, perhaps there is something specific to scala that I am not seeing here...